tools/tm-bench/main.go

```diff
@@ -93,6 +95,19 @@ Examples:
 		"broadcast_tx_"+broadcastTxMethod,
 	)
 
+	//catch Interrupt and quit tm-bench
+	go func() {
+		c := make(chan os.Signal, 1)
+		signal.Notify(c, os.Interrupt, syscall.SIGTERM)
+		for sig := range c {
+			fmt.Printf("captured %v, exiting...\n", sig)
+			for _, t := range transacters {
+				t.Stop()
+			}
+			os.Exit(1)
```

Comment:
Is it an error if the user quits on purpose? Would challenge that and rethink if the binary exit with an error code.